assets/js/vue/FileManager.vue

```diff
@@ -7,28 +7,22 @@
             :close-on-outside-click="true"
             :show-title="true"
             :width="400"
-            :height="400"
-            title="What's New?">
+            :height="350"
+            title="New Uploader!">
             <template>
                 <div id="textBlock">
+                    <h3 style="margin-top: 0">
+                        You can now upload multiple files/folders, add files to an existing dataset, or replace/delete files already uploaded. New features include:
```

Comment:
I would put on "New features include" on a new line, but we'll have the PO decide.